### PR TITLE
setup sorting of widgets

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #:*
https://github.com/bcgov/met-public/issues/<Put the github issue number here>

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/services/widgetService/index.tsx [hunk 1]
```diff
@@ -73,3 +74,22 @@ export const removeWidget = async (engagement_id: number, widget_id: number): Pr
...
     }
 };
+
+export const sortWidgets = async (engagement_id: number, sort_index: number): Promise<Widget> => {
+    try {
+        const url = replaceAllInURL({
+            URL: Endpoints.Widgets.SORT,
+            params: {
+                engagement_id: String(engagement_id),
+                sort_index: String(sort_index),
```
Comment (tiagograf-AOT) [inline met-web/src/services/widgetService/index.tsx:84]: It looks to me that its missing the widget id that is being sorted?!

File: met-web/src/services/widgetService/index.tsx [hunk 2]
```diff
@@ -73,3 +74,22 @@ export const removeWidget = async (engagement_id: number, widget_id: number): Pr
         return Promise.reject(err);
     }
 };
+
+export const sortWidgets = async (engagement_id: number, sort_index: number): Promise<Widget> => {
+    try {
```
Comment (jadmsaadaot) [inline met-web/src/services/widgetService/index.tsx:79]: A bit confused about what will this call perform with the given params. How will the proper widgets sort_index be updated

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Ambiguity around sortWidgets method in widgetService

File: met-web/src/services/widgetService/index.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/services/widgetService/index.tsx:84]: its not in the back end may need some updates in the sorting method

File: met-web/src/services/widgetService/index.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/services/widgetService/index.tsx:84]: updated with the widget index

File: met-web/src/services/widgetService/index.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/services/widgetService/index.tsx]: resolved with the widget index

**Pull request closed without merging**